bin/interpret/analyzefiles.py

```diff
@@ -91,7 +91,7 @@ def plotChroms(sample, outDir, lowessData, refArray, chromList):
 	
 	
 #function to calculate cell summary stats
-def getSummaryStats(cnvs, gender, chromList, chromSizes):
+def getSummaryStats(cnvs, gender, chromList, chromSizes,noCNVs=False):
```

Comment:
```suggestion
def getSummaryStats(cnvs, gender, chromList, chromSizes, noCNVs=False):
```
just for style consistency